include/mapnik/text/scrptrun.hpp

```diff
@@ -17,12 +17,16 @@
 #ifndef __SCRPTRUN_H
 #define __SCRPTRUN_H
 
+#include <mapnik/config.hpp>
 #pragma GCC diagnostic push
 #include <mapnik/warning_ignore.hpp>
 #include <unicode/utypes.h>
 #include <unicode/uobject.h>
 #include <unicode/uscript.h>
 #pragma GCC diagnostic pop
+#include <vector>
+
+const unsigned int STACK_SIZE = 1 << 7; // 2^n
```

Comment:
this doesn't belong to global scope (and is perhaps unnecessarily large)

Reply:
What do you think is a reasonable starting size would be here @lightmare ? How about `16` ? Btw, this code including stack size is copy/pasted from ICU.

Reply:
I was just guessing they chose a large value to cover even crazy cases, because the size was fixed. Most text passing through mapnik are short labels, anyway, so 16 sounds reasonable to me.

Ad copy/paste: I noticed the file comment, was going to submit separate issue/PR. The link to license at icu-project.org is dead, now it's [at github](https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE); mere link is not sufficient to comply with the license, though:
_"... provided that the above
copyright notice(s) and this permission notice appear in all copies of
the Software and that both the above copyright notice(s) and this
permission notice appear in supporting documentation."_

Reply:
> I was just guessing they chose a large value to cover even crazy cases, because the size was fixed. Most text passing through mapnik are short labels, anyway, so 16 sounds reasonable to me.

sounds good to me too.

> Ad copy/paste: I noticed the file comment, was going to submit separate issue/PR. The link to license at icu-project.org is dead, now it's [at github](https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE); mere link is not sufficient to comply with the license, though:
> _"... provided that the above copyright notice(s) and this permission notice appear in all copies of the Software and that both the above copyright notice(s) and this permission notice appear in supporting documentation."_

yes, pls ^